homeassistant/components/mqtt/fan.py

```diff
@@ -58,6 +79,9 @@
 DEFAULT_PAYLOAD_ON = "ON"
 DEFAULT_PAYLOAD_OFF = "OFF"
 DEFAULT_OPTIMISTIC = False
+DEFAULT_PRESET_MODES = ["auto"]
```

Comment:
Will remove this definition since it is not used. The default for preset_modes is []